docker/docker-dev/docker-compose.yml

```diff
@@ -35,6 +35,9 @@ services:
      - "8080:80"
     environment:
       GRAPHITE_CLUSTER_SERVERS: metrictank:6060
+      WSGI_PROCESSES: 4
+      WSGI_THREADS: 25
+      GRAPHITE_STATSD_HOST: statsdaemon
```

Comment:
what i mean is, the reason behind the different docker environments, is that they all have something unique to them, and usually relating to MT. eg: 
* docker-standard : standard docker env. 1 graphite, 1 MT, official binaries and stock configs
* docker-dev: same, but custom MT binary and MT config files mounted in (though the defaults are what is in git, so equivalent to docker-standard by default, which makes it easy to use `git diff` to see the difference)
* docker-cluster : a cluster of MT's

So I don't think we should commit any changes that make the graphite in docker-dev behave any different from any other of the docker environments. maybe those settings be set so that they apply to all our docker environments